code/__DEFINES/weapons.dm

```diff
@@ -57,6 +57,8 @@
 #define FULL_AUTO_600_NOLOSS		list(mode_name = "full auto",    mode_desc = "600 rounds per minute",   mode_type = /datum/firemode/automatic, fire_delay = 2  , icon="auto")
 #define FULL_AUTO_800_NOLOSS		list(mode_name = "fuller auto",  mode_desc = "800 rounds per minute",   mode_type = /datum/firemode/automatic, fire_delay = 1,   icon="auto")
 
+#define FULL_LMG_1		list(mode_name = "suppression",    mode_desc = "spray and pray around 700 rounds per minute",   mode_type = /datum/firemode/automatic, fire_delay = 1.5, icon="auto", damage_mult_add = -0.2)
+#define FULL_LMG_2		list(mode_name = "quicker suppression",    mode_desc = "spray and pray around 800 rounds per minute",   mode_type = /datum/firemode/automatic, fire_delay = 1, icon="auto", damage_mult_add = -0.2)
```

Comment:
```suggestion
```
this is functionally an exact copy of `FULL_AUTO_800`, no reason for it to exist